controller_manager/scripts/controller_group

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Provides quick access to the services exposed by controller_manager_msgs
+import rospy
+from argparse import ArgumentParser
+from controller_manager import controller_manager_interface
+from controller_manager_msgs.srv import ListControllers, ListControllersRequest
+
+
+def _read_group(group):
+    groups = rospy.get_param('controller_groups', {})
```

Comment:
Now using relative parameters, so that it respects ROS_NAMESPACE or roslaunch remapping. Confirmed it works.